frontend/src/metabase/lib/query_time.js

```diff
@@ -273,6 +279,8 @@ export function parseFieldBucketing(field, defaultUnit = null) {
   const dimension = FieldDimension.parseMBQLOrWarn(field);
   if (dimension) {
     return dimension.temporalUnit() || defaultUnit;
+  } else if (field?.[0] === "+" && field?.[1]?.[0] === "field") {
```

Comment:
It is quite hard to understand, can you please rewrite this condition in a [self-descriptive way](https://www.notion.so/metabase/FE-Style-Guide-Proposals-Discussion-acdddeb36bd148e592d8be749ff9e4ef#f9be258edecb4896bac15cc5d15ad871)?

Reply:
Fixed